pyperformance/data-files/benchmarks/bm_gc_traversal/run_benchmark.py

```diff
@@ -0,0 +1,37 @@
+import pyperf
+import gc
+
+N_LEVELS = 1000
+
+
+def create_recursive_containers(n_levels):
+
+    current_list = []
+    for n in range(n_levels):
+        new_list = [None] * n
+        for index in range(n):
+            new_list[index] = current_list
+        current_list = new_list
+
+    return current_list
+
+
+def benchamark_collection(loops, n_levels):
+    total_time = 0
+    all_cycles = create_recursive_containers(n_levels)
+    for _ in range(loops):
+        gc.collect()
+        # Main loop to measure
+        t0 = pyperf.perf_counter()
+        collected = gc.collect()
+        total_time += pyperf.perf_counter() - t0
```

Comment:
Just to be sure I understand, the GC objects aren't cleaned up during `gc.collect()` because `all_cycles` is still bound in the locals.  That's why we can re-use it in each loop.  Is that right?

Reply:
Yeah, this test is just benchmarking the gc traversing all those objects again and again so no need to remove them